datasync/chaindatafetcher/kafka/consumer.go

```diff
@@ -261,6 +264,10 @@ func (c *Consumer) updateOffset(buffer [][]*Segment, lastMsg *sarama.ConsumerMes
 func (c *Consumer) ConsumeClaim(cgs sarama.ConsumerGroupSession, cgc sarama.ConsumerGroupClaim) error {
 	var buffer [][]*Segment
 	for msg := range cgc.Messages() {
+		if len(buffer) > c.config.ConsumerBufferSize {
+			return fmt.Errorf("%v [max: %v, current: %v]", bufferOverflowErrorMsg, c.config.ConsumerBufferSize, len(buffer))
```

Comment:
```suggestion
			return fmt.Errorf("%v : rerun with bigger buffer size. [max: %v, current: %v]", bufferOverflowErrorMsg, c.config.ConsumerBufferSize, len(buffer))
```
Maybe an instruction would be helpful?

Reply:
Reflected. c62d77c19254be0193a36005d2c523af89d0f032